pkg/runner/http.go

```diff
@@ -47,6 +47,8 @@ type ReportResult struct {
 	LastErrorMessage string
 }
 
+const DefaultMaxRecvMsgSize = 4194304
```

Comment:
How do you get this value? Is there any document about it? You can put that link here.

Or, if it is possible to get this value from `gRPC` library.

Reply:
See also https://github.com/grpc/grpc-go/blob/d32e66ce27447a0a217464a36fdd3935801c0453/rpc_util.go#L351

Reply:
Thank you for your help. Before submitting my initial PR, I considered retrieving this value from the gRPC library, but I couldn't locate the corresponding setting. It appears that only the maximum message limit for receiving by gRPC clients can be configured. Therefore, I've decided to update the approach: explicitly setting the maximum message limit for receiving upon establishing a gRPC connection (initially set to the default 4MB). Subsequently, I will return this value in the response function of gRPC, allowing for size determination along with the response body.

Reply:
I found this by searching `4 * 1024 * 1024`, and this two value seems to be unexported.

![Screenshot_2024-04-25-17-27-32-196_com.github.android-edit.jpg](https://github.com/LinuxSuRen/api-testing/assets/51873347/e025d416-c57e-4809-98ab-6c710a468ea0)

Reply:
It looks like an internal const. We are not able to use it.